generated/rflx-derivation.ads

```diff
@@ -10,78 +10,10 @@ is
-
-   subtype Modular_Integer is Arrays.Modular_Integer;
-
-   function Extract (Buffer : RFLX.Types.Bytes; Offset : RFLX.Types.Offset) return Modular_Integer renames Arrays.Extract;
-
-   procedure Insert (Value : Modular_Integer; Buffer : in out RFLX.Types.Bytes; Offset : RFLX.Types.Offset) renames Arrays.Insert;
-
-   function Valid (Value : Modular_Integer) return Boolean renames Arrays.Valid;
-
-   function Convert (Value : Modular_Integer) return Modular_Integer renames Arrays.Convert;
-
-   subtype Range_Integer_Base is Arrays.Range_Integer_Base;
-
-   subtype Range_Integer is Arrays.Range_Integer;
-
-   function Extract (Buffer : RFLX.Types.Bytes; Offset : RFLX.Types.Offset) return Range_Integer_Base renames Arrays.Extract;
-
-   procedure Insert (Value : Range_Integer_Base; Buffer : in out RFLX.Types.Bytes; Offset : RFLX.Types.Offset) renames Arrays.Insert;
-
-   function Valid (Value : Range_Integer_Base) return Boolean renames Arrays.Valid;
-
-   function Convert (Value : Range_Integer_Base) return Range_Integer renames Arrays.Convert;
-
-   subtype Enumeration_Base is Arrays.Enumeration_Base;
-
-   subtype Enumeration is Arrays.Enumeration;
-
-   function Extract (Buffer : RFLX.Types.Bytes; Offset : RFLX.Types.Offset) return Enumeration_Base renames Arrays.Extract;
-
-   procedure Insert (Value : Enumeration_Base; Buffer : in out RFLX.Types.Bytes; Offset : RFLX.Types.Offset) renames Arrays.Insert;
-
-   function Valid (Value : Enumeration_Base) return Boolean renames Arrays.Valid;
-
-   function Convert (Enum : Enumeration) return Enumeration_Base renames Arrays.Convert;
-
-   function Convert (Value : Enumeration_Base) return Enumeration renames Arrays.Convert;
-
-   ZERO : constant Enumeration := Arrays.ZERO;
-
-   ONE : constant Enumeration := Arrays.ONE;
-
-   TWO : constant Enumeration := Arrays.TWO;
-
-   subtype AV_Enumeration_Base is Arrays.AV_Enumeration_Base;
-
-   subtype AV_Enumeration_Enum is Arrays.AV_Enumeration_Enum;
-
-   subtype AV_Enumeration is Arrays.AV_Enumeration;
-
-   function Extract (Buffer : RFLX.Types.Bytes; Offset : RFLX.Types.Offset) return AV_Enumeration_Base renames Arrays.Extract;
-
-   procedure Insert (Value : AV_Enumeration_Base; Buffer : in out RFLX.Types.Bytes; Offset : RFLX.Types.Offset) renames Arrays.Insert;
-
-   function Valid (Value : AV_Enumeration_Base) return Boolean renames Arrays.Valid;
-
-   function Convert (Enum : AV_Enumeration_Enum) return AV_Enumeration_Base renames Arrays.Convert;
-
-   function Convert (Enum : AV_Enumeration_Enum) return AV_Enumeration renames Arrays.Convert;
-
-   function Convert (Value : AV_Enumeration_Base) return AV_Enumeration renames Arrays.Convert;
-
-   function Convert (Value : AV_Enumeration) return AV_Enumeration_Base renames Arrays.Convert;
-
-   AV_ZERO : constant AV_Enumeration_Enum := Arrays.AV_ZERO;
-
-   AV_ONE : constant AV_Enumeration_Enum := Arrays.AV_ONE;
-
-   AV_TWO : constant AV_Enumeration_Enum := Arrays.AV_TWO;
```

Comment:
I don't understand why this part disappeared.

Reply:
This was not intended. Fixed by 05d7086.